lua/refactoring/tasks/indent.lua

```diff
@@ -0,0 +1,68 @@
+local Point = require("refactoring.point")
+local Region = require("refactoring.region")
+
+local function indent_block(start_row, end_row, indent_amount)
+    -- TODO: have to be a better way to do this
+    vim.schedule(function()
+        local temp = (end_row + 1) - start_row
+        local i = 0
+        while i < indent_amount do
+            vim.cmd(":" .. start_row)
+            local indent_cmd = "norm!V" .. temp .. "j" .. "v_>"
+            vim.cmd(indent_cmd)
+            i = i + 1
+        end
+    end)
+end
+
+local function reset_cursor(bufnr_shiftwidth, refactor, total_indents)
+    local new_col = bufnr_shiftwidth * total_indents + 1
+    vim.schedule(function()
+        vim.api.nvim_win_set_cursor(refactor.win, {
+            refactor.result_cursor_row,
+            new_col,
+        })
+    end)
+end
+
+local function indent_func_call(refactor)
+    local scope = refactor.scope
+    -- confirm we need indent with func call scope
+    if not refactor.ts:is_indent_scope(scope) then
+        return
+    end
+
+    -- Get node for function call
+    local func_call_node = Point
+        :from_values(refactor.result_cursor_row, 0)
+        :to_ts_node(refactor.ts:get_root())
+
+    -- Get shiftwidth of current bufnr
+    local bufnr_shiftwidth = vim.bo.shiftwidth
+
+    -- Is this correct? Have to test with python classes to be sure
```

Comment:
i think python will be definitely very hard to do this.  The indentation will break tree sitter traversals.

Reply:
Surprisely at this with just this, it works. With Python it be very different, but editing the raw text should actually be easier before it's applied.